src/NStore.BaseSqlPersistence/BaseSqlPersistenceOptions.cs

```diff
@@ -34,14 +36,21 @@ protected BaseSqlPersistenceOptions(INStoreLoggerFactory loggerFactory)
         public abstract string GetCreateTableIfMissingSql();
         public abstract string GetReadAllChunksSql(int limit);
         public abstract string GetChunkByPositionSql();
-        
+
         public abstract string GetRangeSelectChunksSql(
             long upperIndexInclusive,
             long lowerIndexInclusive,
             int limit,
             bool descending
         );
 
+        public abstract string GetRangeMultiplePartitionSelectChunksSql(
+            IEnumerable<string> partitionIdsList,
+            long upperIndexInclusive,

```

Comment:
ordine dei parametri controintuitivo

Reply:
I know, but here we have the other version so I kept ordering, I'l refactor both call.
![image](https://user-images.githubusercontent.com/358545/217510864-91237de8-4a42-470c-abcc-78468a1de455.png)